.github/workflows/build.yaml

```diff
@@ -0,0 +1,64 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    check:
+        runs-on: ubuntu-20.04
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Composer check
+              uses: shivammathur/setup-php@v2
+              run: composer validate --strict --no-check-lock
```

Comment:
The run mus be a separate step than the setup-php. You are missing a `-`

Reply:
Done